reports-model/src/main/java/org/jboss/da/lookup/model/NPMLookupRequest.java

```diff
@@ -0,0 +1,29 @@
+package org.jboss.da.lookup.model;
+
+import java.util.Set;
+
+import org.jboss.da.model.rest.NPMPackage;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import lombok.Builder;
+import lombok.Data;
+import lombok.NonNull;
+import lombok.extern.jackson.Jacksonized;
+
+@Data
+@Jacksonized
+@Builder
+public class NPMLookupRequest {
+
+    @NonNull
+    private final Set<NPMPackage> artifacts;
+
+    @NonNull
+    private final String mode;
+
+    @NonNull
+    @JsonProperty(defaultValue = "false")
+    private final boolean brewPullActive;
```

Comment:
We don't need brewPullActive for NPM (for now), because it is not implemented on Indy side. Brew pull implementation allows to query only Maven artifacts and AFAIK there is no request to implement that for NPM packages.